contrib/core/actions/uuid.yaml

```diff
@@ -0,0 +1,6 @@
+---
+name: uuid
```

Comment:
Ah that's a good point, `uuid_type` it is then. I will put this to bed by implementing this with support for uuid1 and uuid4. `uuid_type` will be optional so it will default to uuid4. I will try to get this done today with associated test(s).